src/app/attachment/save-action/inbox-linshare-attachment-save-action.service.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .factory('inboxLinshareAttachmentSaveActionService', inboxLinshareAttachmentSaveActionService);
```

Comment:
This will cause an issue in Webpack production mode. It should be `inboxLinshareAttachmentSaveActionServiceFactory`